File: Kattis/Python/sibice.py
import math
x = input().split()
n = int(x[0])
w = int(x[1])
h = int(x[2])
for i in range(n):
    t = int(input())
    if (math.sqrt(w*w+h*h) >= t):
        print("DA")
    else:
        print("NE")

File: Kattis/Java/sibice.java

import java.io.*;
import java.util.*;

class sibice {
  public static void main (String [] args) throws IOException {
	 Scanner file = new Scanner(System.in);
	 int n = file.nextInt();
	 int h = file.nextInt();
	 int w = file.nextInt();
	 double diag = Math.sqrt(h * h  + w * w);
	 for (int i = 0; i < n; i++) {
		 int x = file.nextInt();
		 if (x <= diag) {
			 System.out.println("DA");
		 }
		 else {
			 System.out.println("NE");
		 }
	 }
  }
}
